src/controllers/playback/video/index.js

```diff
@@ -95,18 +95,20 @@ export default function (view) {
         }
 
         setTitle(displayItem, parentName);
-        ratingsText.innerHTML = mediaInfo.getPrimaryMediaInfoHtml(displayItem, {
-            officialRating: false,
-            criticRating: true,
-            starRating: true,
-            endsAt: false,
-            year: false,
-            programIndicator: false,
-            runtime: false,
-            subtitles: false,
-            originalAirDate: false,
-            episodeTitle: false
-        });
+        if (userSettings.enableDisplayRatingsInPlayer()) {
+            ratingsText.innerHTML = mediaInfo.getPrimaryMediaInfoHtml(displayItem, {
```

Comment:
Currently, this won't cause a problem because we can't change `enableDisplayRatingsInPlayer` during playback. In general we should fill everything for the current item, as if the settings can change on the fly, so that we don't have problems in the future.